configsuite/schema.py

```diff
@@ -41,17 +41,17 @@ def _check_allownone_type(schema_level):
 
 
 @configsuite.validator_msg(
-    "Non required types must allow None or have non-None default"
+    "A type is not required only if it allows None or have a non-None default"
```

Comment:
Maybe it because it's nearing the end of the working day, but I think this sentence is hard to read! 
Maybe something like: "A type can only be set to not required if it allows None or have a non-None default"?

Reply:
Never mind, I now see how the validator_msg should be a statement that can be either true/false and not a feedback message directly :facepalm: